modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -40,7 +40,7 @@ validParams<RadialReturnStressUpdate>()
 }
 
 RadialReturnStressUpdate::RadialReturnStressUpdate(const InputParameters & parameters)
-  : StressUpdateBase(parameters),
+  : StateUpdateBase(parameters),
     _max_its(parameters.get<unsigned int>("max_iterations")),
     _output_iteration_info(getParam<bool>("output_iteration_info")),
     _output_iteration_info_on_error(getParam<bool>("output_iteration_info_on_error")),
```

Comment:
**Note done**.  compute_full_tangent operator comes from ComputeAdmissibleState and is not known by StateUpdateBase or derived classes.